edi_oca/templates/exchange_mixin_buttons.xml

```diff
@@ -3,7 +3,7 @@
     <template id="edi_exchange_consumer_mixin_buttons">
         <div
             t-att-groups="group"
-            class="alert alert-warning"
+            class="o_mg_alert alert-warning"
```

Comment:
this looks wrong.  The `o_mg_alert` is only for mail_group alerts. Plus, even if you keep it the `alert` class should stay.

Reply:
Yes, you are right. I had a confusion here....
![image](https://github.com/OCA/edi-framework/assets/28590170/7e30b9e8-1d6a-412e-800f-2ec044307675)